admin/javascript/src/components/grid-field/row.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import SilverStripeComponent from 'silverstripe-component';
+
+class GridFieldRowComponent extends SilverStripeComponent {
+
+  render() {
+    return (
+      <li className="grid-field-row-component [ list-group-item ]">{this.props.children}</li>
```

Comment:
@clarkepaul and I created this structure based on some responsive designs that would've relied on quite a bit of JavaScript to implement if we used a html table structure, whereas with ul/li's we can do it all with css + breakpoints to match the designs in mobile, hence the `display: table` in CSS which would be overridden on mobile.